### ISSUE TITLE
Configuration offline/online mode

### ISSUE THREAD

**Issue by shahraz1998** [amosproj/amos2024ss01-xcelerator-demo-app#216]
## User story

1. As a user
2. I want / need to differentiate my dashboard between online and offline mode
3. So that operate the system without siemens connection

## Acceptance Criteria

-     The dashboard should automatically check if it is operated in a certain environment

## Definition of done (DoD)

-     Added only after week 5
-     The same for all features
-     Here goes the project specific part

## DoD general criteria

-     Feature has been fully implemented
-     Feature has been merged into the mainline
-     All acceptance criteria were met
-     Product owner approved features
-     All tests are passing
-     Developers agreed to release
